third_party/blink/renderer/core/dom/events/event_target.cc

```diff
@@ -222,7 +222,11 @@ void EventTargetData::Trace(Visitor* visitor) const {
   visitor->Trace(event_listener_map);
 }
 
-EventTarget::EventTarget() = default;
+EventTarget::EventTarget() {
+  if (recordreplay::AreEventsDisallowed() &&
+      !recordreplay::HasDivergedFromRecording())
+    recordreplay::Warning("[RUN-1735-1764] EventTarget::EventTarget");
```

Comment:
Isn't this warning redundant with others that will happen when the ScriptWrappable superclass tries to get a record/replay id?  If we want to keep it the message needs to be improved.

Reply:
We don't actually warn in the `NewId*` functions during replay. The main reason for that is probably the `DOMURL` objects that we create in `gReplayScript`. `DOMURL` are `ScriptWrappable`, so I moved the warning to exactly one level above that.

For reference - the warning in `CheckNewId`:

```js
if (AreEventsDisallowed()) {
  // IDs can be created when events are disallowed when our own scripts
  // create URL objects. This would be nice to improve.
  if (!IsInReplayCode()) {
    Warning("NewId when not allowed %s", name);
  }
  return false;
}
```

I could add a follow up to replace the 2 places (that I found) where we create `DOMURL` in `gReplayScript` and then move the warning to the `NewId*` functions.